Tests/Foundation/Tests/TestAttributedStringPerformance.swift

```diff
@@ -415,7 +415,7 @@ final class TestAttributedStringPerformance: XCTestCase {
             ("testSetMultipleAttributes", testSetMultipleAttributes),
             ("testEnumerateAttributes", testEnumerateAttributes),
             ("testEnumerateAttributesSlice", testEnumerateAttributesSlice),
-            ("testConvertToNSAS", testConvertToNSAS),
+            ("testConvertToNSAS", testExpectedToFail(testConvertToNSAS, "rdar://83726414")),
```

Comment:
We usually put `/* ⚠️ */` before the disabled test names like
```
/* ⚠️ */ ("test_dataTaskWithHttpInputStream", testExpectedToFail(test_dataTaskWithHttpInputStream, "Flaky test")),
```
Please avoid mentioning radar number in public repo. For this case feel free to just say `Flaky test` or add other additional information.

```suggestion
            /* ⚠️ */  ("testConvertToNSAS", testExpectedToFail(testConvertToNSAS, "Flaky test")),
```